### PR TITLE
fix: height word typo

### PR DESCRIPTION
**Pull request by 0o001**
## Purpose

_Do a quick recap of your work here._

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

**Review by HBelusca — APPROVED** [review]

**Review by katahiromz — APPROVED** [review]

File: modules/rostests/tests/combotst/combotst.c [hunk 1]
```diff
@@ -7,7 +7,7 @@
 * or make this more complex. Also Fonts are not heavily used.
 
 * This source code is in the PUBLIC DOMAIN and has NO WARRANTY.
-* by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
+* by Waldo Alvarez Cañizares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
```
Comment (HBelusca) [inline modules/rostests/tests/combotst/combotst.c:10]: Note that those two files must be encoded in _**UTF-8 without BOM**_

File: modules/rostests/tests/combotst/combotst.c [hunk 2]
```diff
@@ -7,7 +7,7 @@
 * or make this more complex. Also Fonts are not heavily used.
 
 * This source code is in the PUBLIC DOMAIN and has NO WARRANTY.
-* by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
+ * by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
```
Comment (JoachimHenze) [inline modules/rostests/tests/combotst/combotst.c:10]:
```suggestion
* by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
```
rervert that change please!

**Comment by 0o001** [comment]
> Looks good to me. Before merging, could you please un-caps your last name (`UZUN` -> `Uzun`) in [GitHub profile](https://github.com/settings/profile)?
> 
> image

I did. Thank you.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Revert that comment change please. the rest LGTM.

**Pull request merged**